app/presenters/summary/sections/supporting_evidence.rb

```diff
@@ -0,0 +1,36 @@
+module Summary
+  module Sections
+    class SupportingEvidence < Sections::BaseSection
+      def name
+        :supporting_evidence
+      end
+
+      def show?
+        crime_application.documents.present? && super
```

Comment:
As we've defined a `documents` method, we could reuse it here:
`documents.any? && super`